introduction_to_amazon_algorithms/jumpStart_tabular_classification/Amazon_JumpStart_Tabular_Classification_XGBoost_LinearLearner.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>The formatting for "with the maximum probability over others" seems off</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>